src/btree/bt_read.c

```diff
@@ -351,6 +377,16 @@ __las_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
     /* Instantiate the remaining birthmark entries. */
     WT_ERR(__instantiate_birthmarks(session, ref));
 
+    if (total_incr != 0)
+        __wt_cache_page_inmem_incr(session, page, total_incr);
```

Comment:
Why don't we add:
```
if (size == 0)
    return;
```
into `__wt_cache_page_inmem_incr()` and then we'll catch any functions wasting work on useless increments. It's a register compare, so it's basically free. Then we don't to make any explicit checks.

Reply:
Fixed.